app/src/main/java/com/pixlee/pixleeandroidsdk/GridAdapter.java

```diff
@@ -16,13 +17,11 @@
 public class GridAdapter extends RecyclerView.Adapter<GridViewHolder> {
     private ArrayList<PXLPhoto> galleryList;
     private Context context;
-    private ImageLoader imageLoader;
     private SampleActivity saref;
 
     public GridAdapter(Context context, ArrayList<PXLPhoto> galleryList, SampleActivity sa) {
         this.galleryList = galleryList;
         this.context = context;
-        this.imageLoader = PXLClient.getInstance(context).getImageLoader();
```

Comment:
what happens to this imageloader stuff?

Reply:
What are the diff btw Retrofit and Volley?
 - |             | Volley | Retrofit |
    |-------|--------|--------|
   |Caching| Opt-out | Opt-in |
   |Caching bug| Yes  https://github.com/mcxiaoke/android-volley/issues/134| No |
   |Built-in-image-loading support| Yes | No |
   |Auto-Retry| Yes | No |
   |Coroutines Support| No | Yes |
   |Code complexity|High|Low|

How does this affect our caching?
- this version will not store any api result. Instead it will just show anything from the network as it is.

Does the readme need to be changed at all?
 - I forgot to mention this. Sorry for this. ImageLoader is part of Volley. And Retrofit does not provide image-loading-support. However this not a problem because there is a famous library for loading images which is 'Glide' made by Google. I added Glide to our demo app. Then our users can understand what's going on. 
- Would it be better to mention that the network library changed from Volley to Retrofit, and image-loading should be implemented on their side.  And the sample code is suggested on our demo app.

Reply:
When you make a release tag I think it would be good to mention that in the changelog